app/src/main/java/com/hedvig/app/feature/loggedin/service/TabNotificationService.kt

```diff
@@ -1,52 +1,47 @@
 package com.hedvig.app.feature.loggedin.service
 
-import androidx.datastore.core.DataStore
-import androidx.datastore.preferences.core.Preferences
-import androidx.datastore.preferences.core.edit
-import androidx.datastore.preferences.core.stringSetPreferencesKey
 import com.hedvig.app.feature.loggedin.ui.LoggedInTabs
+import com.hedvig.app.service.badge.NotificationBadge
+import com.hedvig.app.service.badge.NotificationBadgeService
+import com.hedvig.app.service.badge.Seen
+import com.hedvig.app.service.badge.isSeen
 import kotlinx.coroutines.flow.Flow
 import kotlinx.coroutines.flow.map
 
 class TabNotificationService(
     private val getCrossSellsUseCase: GetCrossSellsUseCase,
-    private val dataStore: DataStore<Preferences>,
+    private val notificationBadgeService: NotificationBadgeService,
```

Comment:
I'm a little confused - don't `NotificationBadgeService` and `TabNotificationService` kind of deal with the same subject matter? 🤔 Should they be separated, or should they be the same? 🤔